src/App.js

```diff
@@ -1,15 +1,30 @@
+import { Route, Routes } from "react-router-dom";
 import "./App.css";
-import PriSecBtn from "./components/PriSecBtn";
+import ClassesPage from "./components/pages/ClassesPage";
+import ClassSummaryPage from "./components/pages/ClassSummaryPage";
+import CompetitorsPage from "./components/pages/CompetitorsPage";
+import ContestsPage from "./components/pages/ContestsPage";
+import ExcersisesPage from "./components/pages/ExcersisesPage";
 import HomePage from "./components/pages/HomePage";
+import LoginPage from "./components/pages/LoginPage";
+import SingleSummaryPage from "./components/pages/SingleSummaryPage";
 
 function App() {
   return (
     <div className="App">
-      {/* #24/UI/Primary-secondary-btns - put here for test purposes only */}
-      <PriSecBtn primary text="PRIMARY" />
-      <PriSecBtn secondary text="SECONDARY" />
-      {/* ==== */}
-      <HomePage />
+      <Routes>
+        <Route path="/" element={<HomePage />} />
+        <Route path="/login" element={<LoginPage />} />
+        <Route path="/contests" element={<ContestsPage />}>
+          <Route path=":contestId/class" element={<ClassesPage />}>
+            <Route path=":classId" element={<CompetitorsPage />}>
+              <Route path="Woof" element={<ExcersisesPage />} />
+              <Route path="Woof/summary" element={<SingleSummaryPage />} />
```

Comment:
Woof =>:dogId
.then.nested summary

And so this is a place that i am not sure, I mean a different view of the exercises itself, but i think it should be something like path="exercises and then nested "/:exerciseId (if this view is even needed)